ios_ui/MovieRec/Recommendations.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Recommendations.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/19/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import os.log
+import UIKit
+import SwiftyJSON
+
+protocol RecTableDelegate: class {
+    func endLoadingOverlay()
+    func refreshTable()
+}
+
+class Recommendations {
+    var recommendations = [Recommendation]()
+    var timer: DispatchSourceTimer?
+    weak var delegate: RecTableDelegate?
+    
+    enum TrainingStatus: String {
```

Comment:
> I would actually create an enum of some sort that indicates the status and use its rawValue as its type to be more type safe instead of using a raw string.

Probably not changing. Just needs to check if job is done. So are you saying its probably fine to just use a string?